### DISCUSSION TITLE
Nuclear Overhaul icons

### DISCUSSION THREAD

**Discussion by RedRafe** [malcolmriley/unused-renders#3] [General]
Hi,

although I dont have Reddit, I'm reaching out after seeing your post on Reddit of the assets you're kindly gifting the community. First of all, impressive works! We're all flabbergasted at what you collected through hard work in the past year while making the art pieces.

I've been modding Factorio for quite some time, and while looking ad your assets, I found many nuclear-related items that would nicely fit in my mod [Nuclear Overhaul](https://mods.factorio.com/mod/nuclear-overhaul).

If you'd like, and with your permission & credits, maybe I could borrow some of those (i.e. yellowcake) to "upgrade" my poor attempts at recolouring vanilla sprites. I've managed well with code, but graphics I've never grasped much how to craft, and if this was your journey to learn Blender, I'd very much like to give it a shot sometimes as well and hopefully get something even remotely as cool as your crafts :)

What you've done with these renders and the willingness to release them to the community really speaks by itself. But I encourage you to try and finish your mod idea (and I know working on overhauls takes A LOT). Maybe even trying to release a smaller mod, with something simple or the beginning of your idea, could help push through the initial barrier.

If time is the main issue instead, then I understand the uncertainty of the events. In which case, I still encourage you (as others have said on reddit), to publish these assets just as a graphical mod, that other modders can mark as dependencies on their mod and build on top (see [Unified Molecule Library](https://mods.factorio.com/mod/unified-molecule-library), [Krastorio2 Assets](https://mods.factorio.com/mod/Krastorio2Assets), []()... they're basically just a `info.json`, `readme`, `license` and directories with assets, like your repo already).
I'd much prefer to add your mod as dependency, rather than "stealing" your icons and putting them in my mod :) & you'll still have a mod published to come back one day, if/when you'll decide to give the code part a go.

Ultimately, if you like the idea of lending the icons for Nuclear Overhaul, and you'd like to integrate some processes/ideas/features you'd wish into Nuclear Overhaul, i could implement those codewise in the mod as thank you for your awesome work :)

Thank you!
[Rafe](https://mods.factorio.com/user/RedRafe)

**Comment by malcolmriley** [comment]
Hello,

Thank you very much for your kind message and interest in my work.

I appreciate your asking for permission in reusing the sprites, but it truly isn't necessary - permission is already given by the license I chose. Anyone is free to copy, modify, and redistribute the sprites - whether it is for personal or commercial use. Adapting them into a mod falls somewhere in there, and it's equally okay to monetize the project afterwards (such as with crowdfunding or sponsorships and the like).

As far as learning how use Blender, I would say that anybody that can learn how to mod Factorio can definitely learn how to use Blender. I'm not a "real" artist myself, I primarily consider myself to be a (mediocre) programmer, so if I can do it, you can too. I'm currently in the process of writing a guide series on how to produce Blender renders for Factorio mods with the aim that somebody with zero experience would be able to follow along and create some useful images.

With regard to my own mod, time constraint was one of several reasons I've shelved the project. I fully intend to return to it one day, but then again, I fully intend to return to all of my shelved projects. At the very least, with the release of Space Age on the horizon, I'd like to wait and see how the expansion changes the game before resuming work. I've read a few FFF's that imply a much superior implementation of ideas I'd had (if you look through the repository you might be able to piece together which ones). With the other projects I'm attending to in mind, it seemed a bit wasteful to try to cobble together a scripted solution to a problem that might have a built-in solution in the future API.

This last part isn't directed at you specifically - I've seen a similar sentiment frequently, and on the internet we are joined by the ghostly presence of eavesdroppers and future readers. I'd like to close by underscoring the fact that using the sprites in this repository really is free. I don't see it as "lending" or "borrowing" or much less "stealing" to adapt or redistribute these works, not even if you sell it for money afterwards. It is merely an offering, and one can accept or reject it as best suits their needs. One cannot steal what is freely given.

Best of luck with your project, I'm glad to hear that my sprites may be of use.

**Comment by RedRafe** [comment]
Yes - I did get that the license allowed free use, I just preferred to notify you of my intentions beforehand. Still, if you'll come back to modding your ideas, I can safely say there'll be lots of people in this community ready to help and assist, as it also happen with every new modder entering the scene :)

Best of luck to you too, here's the updated image from **Nuclear Overhaul** Yellowcake, Thorium & Fluorite. Thanks again  

![Screenshot from 2024-06-14 10-22-23](https://github.com/malcolmriley/unused-renders/assets/93430988/4ff6f9ab-c713-42d2-b7fc-dfc65e9f647c)

**Reply by malcolmriley** [reply]
Of course! Thank you again for reaching out, it's always a joy to see the sprites in use.
